### PR TITLE
AAE-1502 add script runtime

### PR DESCRIPTION
**Pull request by mteodori**

### CODE REVIEW

File: example-dmn-runtime-service/build.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-DOCKER_IMAGE_NAME=example-dmn-runtime-service
+DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME:-$(basename $PWD)}
```
Comment (ffazzini) [inline example-dmn-runtime-service/build.properties:1]: it is not totally clear to me the whole `${DOCKER_IMAGE_NAME:-$(basename $PWD)}`
Is it necessary? Can i be simplified?

File: example-script-runtime-service/project/scripts/example.bin [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('Hello world');
```
Comment (ffazzini) [inline example-script-runtime-service/project/scripts/example.bin:1]: what is the purpose of this file?

File: example-script-runtime-service/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+``` bash
+docker run -it -p 127.0.0.1:8080:8080 \
+  --env SCRIPTCONFIGURATION_SCRIPTSDEFINITIONSDIRECTORYPATH=file:/maven/scripts/ \
+  --env PROJECT_MANIFEST_FILE_PATH=file:/maven/project.json \
+  --env KEYCLOAK_AUTHSERVERURL=https://identity.***/auth \
+  --env SPRING_DATASOURCE_URL=jdbc:postgresql://***.us-east-1.rds.amazonaws.com:5432/external-db-app \
+  --env SPRING_DATASOURCE_USERNAME=*** \
+  --env SPRING_DATASOURCE_PASSWORD=*** \
+  --env SPRING_RABBITMQ_HOST=*** \
```
Comment (ffazzini) [inline example-script-runtime-service/README.md:23]: i am not sure we have a queue for this runtime service, @mmartinadanx ?

File: example-script-runtime-service/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,4 @@
+FROM quay.io/alfresco/alfresco-script-app-runtime:develop
+
+COPY project/scripts /maven/scripts
+COPY project/project.json /maven/project.json
```
Comment (ffazzini) [inline example-script-runtime-service/Dockerfile]: "COPY project/project.json /maven/project.json" is one of the changes with the upgrade in m6, we won't need this project.json file. We can remove it from here, and i need to remove it later from the other projects except process-runtiime

File: example-script-runtime-service/project/project.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (ffazzini) [inline example-script-runtime-service/project/project.json]: this is one of the changes with the upgrade in m6, we won't need this file. We can remove it from here, and i need to remove it later from the other projects except process-runtiime

File: example-dmn-runtime-service/build.properties (hunk 1 above)
Comment (mteodori) [reply example-dmn-runtime-service/build.properties:1]: it is actually better as it takes the name from the directory and it is the same everywhere

File: example-script-runtime-service/project/scripts/example.bin (hunk 2 above)
Comment (mteodori) [reply example-script-runtime-service/project/scripts/example.bin:1]: an example script to drop in

File: example-script-runtime-service/Dockerfile (hunk 4 above)
Comment (mteodori) [reply example-script-runtime-service/Dockerfile]: ok I just copied from DMN, can remove this

File: example-dmn-runtime-service/build.properties (hunk 1 above)
Comment (ffazzini) [reply example-dmn-runtime-service/build.properties:1]: It is better if the customer would use this exactly like it is, but i doubt the use the same trick of the folder name in production.

So on my opinion i am not sure that it is better, but definitely it is not clearer as there is more scripting and the reader needs to get the logic you wanted to apply

so better to leave a simple string, then the customers in their own pipelines can customise as they wish

File: example-script-runtime-service/README.md (hunk 3 above)
Comment (mteodori) [reply example-script-runtime-service/README.md:23]: I can confirm as script runtime is a connector

File: example-dmn-runtime-service/build.properties (hunk 1 above)
Comment (mteodori) [reply example-dmn-runtime-service/build.properties:1]: I would prefer to keep it as is so you don't forget to update it

**Review by ffazzini — APPROVED** [review]

**Pull request merged**